jre_emul/android/platform/libcore/luni/src/test/java/libcore/java/util/jar/JarFileTest.java

```diff
@@ -0,0 +1,1067 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package libcore.java.util.jar;
+
+//import java.io.ByteArrayOutputStream;
```

Comment:
Add `/* J2ObjC Removed... */` to make future upgrades easier? Same for other places in this file and in other files.

Reply:
I'm not sure that's necessary if we instead move the "feature not supported on iOS" comment at the start of the class here.

Reply:
moved the comment to the start of class